apps/dokploy/templates/templates.ts

```diff
@@ -497,4 +497,19 @@ export const templates: TemplateData[] = [
 		tags: ["self-hosted", "storage"],
 		load: () => import("./gitea/index").then((m) => m.generate),
 	},
+	{
+		id: "roundcube",
+		name: "Roundcube",
+		version: "1.6.9",
+		description:
+			"Free and open source webmail software for the masses, written in PHP.",
+		logo: "roundcube.svg",
+		links: {
+			github: "https://github.com/roundcube/roundcubemail",
+			website: "https://roundcube.net/",
+			docs: "https://roundcube.net/about/",
+		},
+		tags: ["self-hosted", "mail", "webmail"],
```

Comment:
we have tag "email", i suggest a change from "mail" -> "email"

Reply:
Should be OK in the latest commit, do we keep webmail as a "new" tag?
Thanks!

Reply:
Yep, I think what "webmail" can be a new tag. I suggest a change for "email" because we have another templates with this tag.

Reply:
Yep you're 100% correct. I should've checked on my local what labels were available... 😄